src/app.js

```diff
@@ -13,5 +14,8 @@ app.engine('hbs', exphbs({
 
 app.use(express.static(path.join(__dirname, '..', 'public')));
 app.set('port', process.env.PORT || 3000);
+app.use(controllers);
+app.use(serverError);
```

Comment:
Why exporting `serveError` as a separate variable when it's part of `controllers`?